ugandaemr_mobile/download-patients.md

```diff
@@ -0,0 +1,26 @@
+
+## Downloading Patient In UgandaEMR Mobile
+1. To Download Patients Tap Download Patient on the menu screen.
+2. Select Appointment Start and End Dates for patients with appointments to download, after Tap on the Download Patient Button as shown in the figure below. ![Download Patient Screen](.gitbook/assets/download.png)
+   1. After downloading patients a dialog will popup as shown in the figure below.
+      ![Download Finished Screen](assets/download-finish.png)
+## Find Patients In UgandaEMR Mobile
+1. On the menu screen Tap Find Patient, a screen below will be seen.
+   Enter appointment date or vist date, Patient Name or ART Number and Tap search button to assess the patient. 
+   ![Find Patient Screen](assets/search.png)
+if patient is found a screen below will be show to start assessing the patient. ![Search Found Screen](images/search-found.png)
+
+## Work Summary
+1. To assess a patient in UgandaEMR mobile app Tap Work Summary from the menu screen or home screen, Select visit date for patients with appointments. use the filter to navigate through different options. ![work summary Screen](assets/img15.jpg)
+2. On selecting the visit date is selected, alist of downloaded patients will be seen as shown in the figure below. to assess a patient Tap on the patient record. ![Assess Patient button](.gitbook/assets/img16.jpg)
+3. To assess a patient Tap Assess Patient to start an assessment as shown ![Assessment Screen](images/client_selected.png)
+  NOTE:
+   If the patient is  represented Tap Yes button and Dispense drugs to the Patient otherwise click YES to start the assessment.
+   start assessing the patient. 
+## Drug Dispensing 
+1 . Tap on the drug or combination from the Drug administration tables as shown below to enter pills dispensed and duration in days. ![Drug administration Screen](images/drug_dispensing.png)
```

Comment:
@solemabrothers Remove the empty spaces between the number and the dots, that is special meaning to that